azurelinuxagent/ga/update.py

```diff
@@ -315,18 +315,22 @@ def run(self, debug=False):
             logger.info("OS: {0} {1}", DISTRO_NAME, DISTRO_VERSION)
             logger.info("Python: {0}.{1}.{2}", PY_VERSION_MAJOR, PY_VERSION_MINOR, PY_VERSION_MICRO)
 
+            vm_arch = self._get_vm_arch()
+            logger.info("CPU Arch: {0}", vm_arch)
```

Comment:
do we need to log here? we are already logging below

Reply:
ah, i see, this was intentional, following the lead of OS & python. OK as is, I guess.